### PR TITLE
PWGGA/GammaConv: reduce number of casts for cutArrays

### PR DESCRIPTION
**Pull request by sstiefel19**

### CODE REVIEW

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx [hunk 1]
```diff
@@ -764,8 +770,8 @@ void AliAnalysisTaskGammaConvV1::InitBack(){
         fBGHandlerRP[iCut] = NULL;
       } else if(((AliConversionMesonCuts*)fMesonCutArray->At(iCut))->BackgroundHandlerType() != 2){
         fBGHandlerRP[iCut] = new AliConversionAODBGHandlerRP(
-                                  ((AliConvEventCuts*)fEventCutArray->At(fiCut))->IsHeavyIon(),
-                                  ((AliConversionMesonCuts*)fMesonCutArray->At(fiCut))->UseTrackMultiplicity(),
+                                  ((AliConvEventCuts*)fEventCutArray->At(fiCut))->IsHeavyIon(),  // SFS fiCut, is this intended?
```
Comment (sstiefel19) [inline PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx:773]: Id assume fiCut is a typo here and it is meant to be iCut. Do you agree?

Comment (dmuhlhei) [reply PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx:773]: Yes, it has to be iCut - please fix it

Comment (sstiefel19) [reply PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx]: done

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**